slackbot/dispatcher.py

```diff
@@ -98,7 +98,7 @@ def _default_reply(self, msg):
         default_reply = [
             u'Bad command "%s", You can ask me one of the following questions:\n' % msg['text'],
         ]
-        default_reply += [u'    • `%s`' % str(p.pattern) for p in self._plugins.commands['respond_to'].iterkeys()]
+        default_reply += [u'    • `%s`' % unicode(p.pattern) for p in self._plugins.commands['respond_to'].iterkeys()]
```

Comment:
@dangra good point about `format`, I just noticed that the original author uses `%s'-style formatting everywhere (instead of str.format) in the project, and I wanted to make as few changes as possible.